### ISSUE TITLE
Routing broken after upgrade

### ISSUE THREAD

**Issue by Najarana** [rocjs/roc-package-web-app-react#87]
After upgrading from 1.0.0-beta3 to ^1.0.0 when first trying to access a route that requires auth. If we first go to /login the other routes will work properly.  Meaning that if we fail auth => replace on the first route we enter we will get this error. 

We have tried with and without useDefaultRoutes. 

```
Uncaught TypeError: Cannot read property 'getCurrentLocation' of undefined
    at Object.createTransitionManager (Router.js:112)
    at Object.componentWillMount (Router.js:119)
    at ReactCompositeComponent.js:345
    at measureLifeCyclePerf (ReactCompositeComponent.js:73)
    at ReactCompositeComponentWrapper.performInitialMount (ReactCompositeComponent.js:344)
    at ReactCompositeComponentWrapper.mountComponent (ReactCompositeComponent.js:255)
    at Object.mountComponent (ReactReconciler.js:43)
    at ReactCompositeComponentWrapper.performInitialMount (ReactCompositeComponent.js:368)
    at ReactCompositeComponentWrapper.mountComponent (ReactCompositeComponent.js:255)
```

This is our router:

```
export default () => {
  function requireAuth(nextState, replace) {
    debugger;
    if (!session.isTokenValid()) {
      replace({
        pathname: '/login',
        state: { nextPathname: nextState.location.pathname },
      });
    }
  }


  const router = (
    <Router history={browserHistory}>
      <Route path="/" component={ App }>
        <IndexRoute component={ FleetMapPage }
          onEnter={(nextState, replace) => { requireAuth(nextState, replace); }}
        />
        <Route component={ SupportPage } path="/support(/search/:query)"
          onEnter={(nextState, replace) => { requireAuth(nextState, replace); }} />
        <Route component={ EventPage } path="/event/"
          onEnter={(nextState, replace) => { requireAuth(nextState, replace); }} />
        <Route component={ ContactPage } path="/contact/"
          onEnter={(nextState, replace) => { requireAuth(nextState, replace); }} />
        <Route component={ PoolPage } path="/pools(:query)"
          onEnter={(nextState, replace) => { requireAuth(nextState, replace); }} />
        <Route component={ LoginPage } path="/login" />
      </Route>
    </Router>
  );
  return router;
};
```
Any help would be greatly appreciated

**Comment by dlmr** [comment]
This is a known error and we are working on fixing it and it should be available soon.

For now I would recommend to downgrade to the latest beta version in the meantime, `1.0.0-beta.17`.

**Comment by dlmr** [comment]
This has been solved in [`roc-package-web-app-react@1.0.1`](https://github.com/rocjs/extensions/releases/tag/roc-package-web-app-react%401.0.1).
